hips/tiles/surveys.py

```diff
@@ -127,7 +132,12 @@ def tile_format(self) -> str:
     @property
     def base_url(self) -> str:
```

Comment:
@tboch Can you please list a survey which is missing both `hips_service_url` and `moc_access_url`.